package.json

```diff
@@ -27,16 +27,16 @@
     "db:migrate-up-all": "npm run db:migrate",
     "db:migrate-down": "./src/server/database/migrate.sh down",
     "db:migrate-down-all": "./src/server/database/migrate.sh down-all",
-    "lint": "eslint 'src/**/*.js' 'tests/*.js' 'webpack/*.js'",
+    "lint": "eslint --fix 'src/**/*.js' 'tests/*.js' 'webpack/*.js'",
     "profile": "NODE_ENV=production webpack --config ./webpack/prod.babel.js --profile --json > stats.json"
   },
   "pre-commit": [],
   "devDependencies": {
-    "assets-webpack-plugin": "3.4.0",
+    "assets-webpack-plugin": "git://github.com/mattkrick/assets-webpack-plugin.git/#master",
```

Comment:
radio silence for my PR: https://github.com/kossnocorp/assets-webpack-plugin/pull/45